index.html

```diff
@@ -1,196 +1,273 @@
+          </p>
+          <ul class="code-btn-container">
+            <li class="code-btn">HTML</li>
+            <li class="code-btn">CSS</li>
+            <li class="code-btn">Javascript</li>
+          </ul>
+          <button class="action" type="button">See Project</button>
+        </li>
+      </ul>
+    </section>
+    <section class="aboutme">
+      <h2 class="aboutme-title">About me</h2>
+      <p class="aboutme-text">
+        Hello I’m a software developer! I can help you build a product , feature
+        or website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+      <button class="btn" type="button">Get My Resume</button>
+      <ul class="skills">
+        <li class="skill">
+          <div class="division">
+            <div class="shape1"></div>
+            <span class="skill-name">Language</span>
+          </div>
+          <ul class="child-ul">
+            <li class="child-li">Javascript</li>
+            <li class="child-li">Ruby</li>
+            <li class="child-li">HTML</li>
+            <li class="child-li">CSS</li>
+          </ul>
+        </li>
+        <li class="skill">
+          <div class="division">
+            <div class="shape2"></div>
+            <span class="skill-name">Frameworks</span>
+          </div>
+          <ul class="child-ul">
+            <li class="child-li">Bootstrap</li>
+            <li class="child-li">Ruby on Rails</li>
+            <li class="child-li">RSPec</li>
+            <li class="child-li">CapyBara</li>
+            <li class="child-li">Selenium</li>
+          </ul>
+        </li>
+        <li class="skill">
+          <div class="division">
+            <div class="shape3"></div>
+            <span class="skill-name">Skills</span>
+          </div>
+          <ul class="child-ul">
+            <li class="child-li">Codekit</li>
+            <li class="child-li">Github</li>
+            <li class="child-li">Codepen</li>
+            <li class="child-li">Gitlab</li>
+            <li class="child-li">Terminal</li>
+          </ul>
+        </li>
+      </ul>
+    </section>
+    <section class="contact-form">
+      <h2 class="contact-text">
+        I'm always interested in hearing about new projects, so if you'd like to
+        chat please get in touch.
+      </h2>
+      <form class="form" action="https://formspree.io/f/mnqrnjbr" method="POST">
+        <input
+          class="input"
+          type="text"
+          maxlength="30"
+          name="name"
+          placeholder="Full Name"
+          required
+        />
+        <input
+          class="input"
+          type="email"
+          name="email"
+          placeholder="Email address"
+          required
+        />
+        <textarea
+          class="input textarea"
+          name="message"
+          maxlength="500"
+          placeholder="Enter text here"
+          required
+        ></textarea>
```

Comment:
- [ ] it would be better if you change the background color of the `textarea` in order to match the Figma design 
<img width="912" alt="Screen Shot 2022-08-25 at 13 14 44" src="https://user-images.githubusercontent.com/61790587/186662495-347e5f88-0c3f-4681-873e-e340003b1b21.png">